cypress/support/commands.js

```diff
@@ -22,4 +22,15 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('createWorker', (worker) => {
```

Comment:
This command does not have any error handling. It would be better to add some error handling for the case when worker object does not contain necessary fields.